src/Riok.Mapperly/Descriptors/SymbolAccessor.cs

```diff
@@ -185,6 +185,11 @@ private IEnumerable<ISymbol> GetAllMembersCore(ITypeSymbol symbol)
 
     private IEnumerable<IMappableMember> GetAllAccessibleMappableMembersCore(ITypeSymbol symbol)
     {
+        if (symbol.IsTupleType && symbol is INamedTypeSymbol namedType)
+        {
+            return namedType.TupleElements.Where(x => x is { Kind: SymbolKind.Field }).Select(MappableMember.Create).WhereNotNull();
```

Comment:
Is the where part here needed, aren't these filtered by create+whereNotNull anyway? Does it bring performance benefits? If so could we move it to the top of the create method and return null? Also for single field assertions this code base usually uses the `x.Kind == SymbolKind.Field` style instead of pattern matching.